docs/static/releasenotes.asciidoc

```diff
@@ -172,6 +196,14 @@ If this situation occurs, {ls} may fail to start or some plugins may use a malfo
 Check out issue https://github.com/elastic/logstash/issues/16437[#16437] for details.
 +
 Workaround: Downgrade to {ls} 8.15.0, or temporarily avoid using environment and keystore variable references.
+* **{ls} can fail to shut down under some circumstances when you are using <<pipeline-to-pipeline>>.**
+Check out issue https://github.com/elastic/logstash/issues/16657[#16657] for details.
++
```

Comment:
```suggestion

```

Reply:
I _think_ this one is necessary because it's in a bulleted list?

Reply:
I suggested removing the bullet because it was a single item. When the bullet went away, the `+` needed to go away, too, to avoid it rendering. 

In the most recent review, the suggestion was to remove the `+`, but leave the line so that it would render as two paragraphs.

Reply:
Ideally, the `+` should go, but the line should remain to force rendering into two distinct paragraphs.

Reply:
<img width="864" alt="Screen Shot 2024-11-11 at 4 38 12 PM" src="https://github.com/user-attachments/assets/63e6363e-149d-49f0-a538-36ed94141fcb">